src/index.html

```diff
@@ -10,13 +10,98 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo header w Logo</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          class="img"
+          src="../src/images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
```

Comment:
You are not using BEM in any class in this code.
Here is an example of what nav should be named:
```
<nav class="header__navbar navbar">
    <ul class="navbar__links">
        <li class="navbar__option">
            <a class="navbar__link">
 ....
```

Reply:
Thanks, just fixed the code and sent a review request.